tests/Integration/Behaviour/Features/Scenario/Attribute/attribute_management.feature

```diff
@@ -0,0 +1,48 @@
+# ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s attribute
+@restore-all-tables-before-feature
+@attribute-management
+Feature: Attribute group management
+  PrestaShop allows BO users to manage product attribute groups
+  As a BO user
+  I must be able to create, edit and delete product features
+
+  Background:
+    Given shop "shop1" with name "test_shop" exists
```

Comment:
As suggested above we should add some tests regarding the shop association, like it was done for those tests:
https://github.com/PrestaShop/PrestaShop/blob/6aebb598019e1aca695dc3bd15573746e32151fd/tests/Integration/Behaviour/Features/Scenario/CustomerGroup/customer_group_management.feature